openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -103,12 +102,12 @@ <h4 class="question__title ui-slidable__control">
 
                     {% if peer_assessments %}
                         <form class="submission__feedback submission__feedback step__content__section ui-slidable__container is--showing" method="post">
-                            <h3 class="submission__feedback__title ui-slidable__control">
+                            <div class="submission__feedback__title ui-slidable__control">
                                 <button class="ui-slidable" aria-expanded="true" id="oa_grade_feedback_{{ xblock_id }}" aria-controls="oa_grade_feedback_{{ xblock_id }}__content">
                                     <span class="icon fa fa-caret-right" aria-hidden="true"></span>
                                     <span class="submission__feedback__title__copy">{% trans "Provide Feedback on Peer Assessments" %}</span>
                                 </button>
-                            </h3>
+                            </div>
```

Comment:
@staubina Please make the title "Provide Feedback on Peer Assessments" sentence case, for consistency with equivalent title above (e.g. "Additional comments on your response".